old-ui/app/css/rif.css

```diff
@@ -0,0 +1,65 @@
+.body {
```

Comment:
This needs to be in a file inside a folder called rif, something like old-ui/app/css/rif/feature-name.css or something like that, if we start to add css styles to only one file by the time we end working on this we will have a huge file of css with a lot of lines. It's hard to maintain, we should put that in different files.

Reply:
You right, i'm working on this!

Reply:
Check the new structure, as fs cant import a directory on windows/mac we need to import each .css individually.